File: 2.server/src/main/java/ru/vitaliy/petrov/server/forms/requests/documents/AdministrativeOffenceCaseInvestigationUpdateRequest.java
package ru.vitaliy.petrov.server.forms.requests.documents;

import lombok.Data;

import javax.validation.constraints.NotEmpty;
import javax.validation.constraints.NotNull;
import java.sql.Date;
import java.sql.Time;

@Data
public class AdministrativeOffenceCaseInvestigationUpdateRequest {

    @NotNull
    private final Long documentID;

    @NotNull
    private final Date updatedDateOfFill;

    @NotNull
    private final Time updatedTimeOfFill;

    @NotNull
    @NotEmpty
    private final String updatedPlaceOfFill;

    @NotNull
    @NotEmpty
    private final String updatedInvestigationReason;

    @NotNull
    @NotEmpty
    private final String updatedLawViolationInfo;

    @NotNull
    private final Long carAccidentEntityID;
}

File: client/app/src/main/java/com/petrov/vitaliy/caraccidentapp/domain/models/requests/documents/AdministrativeOffenceCaseInvestigationUpdateRequest.kt
package com.petrov.vitaliy.caraccidentapp.domain.models.requests.documents

import java.sql.Date
import java.sql.Time

data class AdministrativeOffenceCaseInvestigationUpdateRequest(
    val documentID: Long,
    val updatedDateOfFill: Date,
    val updatedTimeOfFill: Time,
    val updatedPlaceOfFill: String,
    val updatedInvestigationReason: String,
    val updatedLawViolationInfo: String,
    val carAccidentEntityID: Long
)
